src/components/constraints/ConstraintListItem.svelte

```diff
@@ -76,9 +117,51 @@
             <VisibleHideIcon />
           {/if}
         </button>
+        <select
```

Comment:
Seeing an issue where the versions are not automatically updating in the dropdown after created, requires refreshing the page.